rocAL/rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -23,10 +23,8 @@
 from amd.rocal.pipeline import Pipeline
 
 def image(*inputs, user_feature_key_map=None, path='', file_root='', annotations_file='', shard_id=0, num_shards=1, random_shuffle=False, 
-          affine=True, bytes_per_sample_hint=0, cache_batch_copy=True, cache_debug=False, cache_size=0, cache_threshold=0, cache_type='', 
-          device_memory_padding=16777216, host_memory_padding=8388608, hybrid_huffman_threshold=1000000, output_type=types.RGB, 
-          decoder_type=types.DECODER_TJPEG, preserve=False, seed=1, split_stages=False, use_chunk_allocator=False, use_fast_idct=False,
-          device=None, decode_size_policy=types.USER_GIVEN_SIZE_ORIG, max_decoded_width=1000, max_decoded_height=1000):
+          output_type=types.RGB, decoder_type=types.DECODER_TJPEG, device=None,
+          decode_size_policy=types.USER_GIVEN_SIZE_ORIG, max_decoded_width=1000, max_decoded_height=1000):
```

Comment:
The default decode size policy used in training is user given size restricted ryt?

Reply:
Yes this type is the same as user given size restricted and we are setting it as the default